cognite_toolkit/_cdf_tk/client/api/charts.py

```diff
@@ -54,6 +54,10 @@ def upsert(self, items: ChartWrite | Sequence[ChartWrite]) -> Chart | ChartList:
                 json=body,
             )
             result.extend([Chart._load(item, cognite_client=self._cognite_client) for item in response.json()["items"]])
+
+        if not result:
```

Comment:
Shouldn't we expect exactly as many results as inputs? Send 10, get 10?

Reply:
I'm not sure, its more a discussion - perhaps in the SDK  / API channels. Clearly for a single item upsert we're enforcing it. So then if we upsert 2, can we rely on 2 coming back.